src/app/utils/firebase.js

```diff
@@ -15,15 +17,32 @@ const FireBaseTools = {
    */
     getProvider: (provider) => {
         switch (provider) {
+
         case 'email':
             return new firebase.auth.EmailAuthProvider();
         case 'google':
             return new firebase.auth.GoogleAuthProvider();
         default:
             throw new Error('Provider is not supported!!!');
         }
+        
     },
 
+
+    getUserCourses :()=>{
+        var userCur=null,id = firebaseAuth.currentUser? firebaseAuth.currentUser.uid: null
+        return rootRef.once('value').then(function (snap) {
```

Comment:
This is perfectly fine right now, but eventually when we read data from Firebase, we'll need to do it through active listeners (without the .once function). Right now, since we only get a single snapshot, we don't see changes made in the DB in real time on the home page without refreshing the page.